viscy/utils/mp_utils.py

```diff
@@ -260,11 +260,16 @@ def get_val_stats(sample_values):
                                         indices
     :return dict meta_row: Dict with intensity data for image
     """
+    p5 = float(np.nanpercentile(sample_values, 5))
+    p95 = float(np.nanpercentile(sample_values, 95))
```

Comment:
We can probably compute `q=[1, 5, 25, 50, 75, 95, 99]` in one percentile (sorting) call here and derive the median and iqr too.